apps/bsp/bulk-scan-payment-processor/demo.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   values:
     java:
-      image: hmctspublic.azurecr.io/bulk-scan/processor:pr-2859-621c68e-20230119112028 #{"$imagepolicy": "flux-system:demo-bulk-scan-processor"}
+      image: hmctspublic.azurecr.io/bulk-scan/processor:pr-2859-621c68e-20230119112028 #{"$imagepolicy": "flux-system:demo-bulk-scan-payment-processor"}
```

Comment:
the image should be:
bulk-scan/payment-processor:pr-880-cca3e94-20230124115908

Reply:
imageAutomation changes this to match new policy once merged.